File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_5.c
#include <stdio.h>

int main() {
	int codigo, quantidade;
	float valor, total;

	printf("Informe o codigo da peca: ");
	scanf("%d", &codigo);

	printf("Informe o valor da peca: ");
	scanf("%d", &valor);

    printf("Informe a quantidade de pecas: ");
    scanf("%d", &quantidade);

    total = quantidade * valor;

    printf("Codigo da peca: %d", codigo);
	printf("Valor total: %.2f", total);
}

File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_5.java
import java.util.Scanner;

public class exercicio_5 {

	public static void main(String[] args) {
		int codigo, quantidade;
		float valor, total;
		Scanner teclado = new Scanner(System.in);

		System.out.print("Informe o codigo da peca: ");
		codigo = teclado.nextInt();

		System.out.print("Informe o valor da peca: ");
		valor = teclado.nextFloat();

		System.out.print("Informe a quantidade de pecas: ");
		quantidade = teclado.nextInt();

		total = quantidade * valor;

		System.out.println("Codigo da peca: " + codigo);
		System.out.println("Valor total: " + total);
		
		teclado.close();
	}

}
